functions/greater-than-ten/greater-than-ten.spec.js

```diff
@@ -0,0 +1,7 @@
+const checkValue = require("./greater-than-ten");
+describe("greater-than-ten", () => {
+  it("should return true if greater 10", () => {

```

Comment:
should also have a case for when its false

Reply:
done :)